### DISCUSSION TITLE
Snapshot retention and Snapshot API json format

### DISCUSSION THREAD

**Discussion by nutslove** [grafana/grafana#51903] [Dashboards, Panels & Visualizations]
Hi,
I have two questions about Snapshot data retention and Snapshot API JSON format.

1. Snapshot data retention
When I choose "Never" Expire, is it stored forever?
Even after data is removed in a backend like VictoriaMetrics?

2. Snapshot API JSON format
I am trying to get snapshots using API as seen at the below URL.
But I don't know what should I write in JSON.
It would be helpful if there are some specific examples.
https://grafana.com/docs/grafana/latest/developers/http_api/snapshot/#snapshot-api

I appreciate any help you can provide.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
